code/modules/mob/living/simple_animal/abnormality/aleph/army_in_black.dm

```diff
@@ -0,0 +1,498 @@
+#define STATUS_EFFECT_PROTECTION /datum/status_effect/protection
+GLOBAL_LIST_EMPTY(army)
+/mob/living/simple_animal/hostile/abnormality/army
+	name = "Army in Black"
+	desc = "The color of the human heart is pink, and by wearing the same color, we can blend in with people’s minds."
+	icon = 'ModularTegustation/Teguicons/64x64.dmi'
+	icon_state = "armyinpink"
+	icon_living = "armyinpink"
+	icon_dead = "armyinpink_heart"
+	pixel_x = -16
+	base_pixel_x = -16
+
+	//*--Suppression info--*
+	maxHealth = 450
+	health = 450
+	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 1.2, WHITE_DAMAGE = 0.6, BLACK_DAMAGE = 1.0, PALE_DAMAGE = 0.8)//the same as hostile mobs, ez records
+	speed = 1//abno itself shouldn't have any stats
+	move_to_delay = 5
```

Comment:
Instead of manually setting speed, rely on UpdateSpeed() to change the value of `speed` to it's appropriate `move_to_delay` equivalent.